### PR TITLE
Added translation for Chinese Simplified

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: share/src/main/resources/assets/wdl/lang/zh_CN.lang [hunk 1]
```diff
@@ -0,0 +1,523 @@
...
+# For description messages in GUIs, you are also allowed to use \n to indicate a new line.
+# This cannot be escaped.
+#
+# Additionally, Minecraft's §-based color codes may be used.  See
+# https://minecraft.gamepedia.com/Formatting_codes for more info.
+
+# Translator name, preferably in the form of 'Translated to %s by <your name>'.
+# '%s' will be replaced with the language name automatically.
+# If left blank, it will not be displayed.
+wdl.translatorCredit=
```
Comment (Pokechu22) [inline share/src/main/resources/assets/wdl/lang/zh_CN.lang]: Do you want to fill this one in?  It's displayed in the about screen.

File: share/src/main/resources/assets/wdl/lang/zh_CN.lang [hunk 2]
```diff
@@ -0,0 +1,523 @@
...
+wdl.gui.player.position.keep=玩家位置：不做改变
+wdl.gui.player.position.xyz=玩家位置：
+wdl.gui.player.setPositionToCurrentPosition=当前位置
+wdl.gui.player.health.description=此选项修改在下载的世界中玩家的生命。\n\n§l不做改变§r：使用下载结束时的生命值。\n§l加满§r：将玩家生命设置为最大值。
+wdl.gui.player.hunger.description=此选项修改在下载的世界中玩家的饥饿。\n\n§l不做改变§r：使用下载结束时的饥饿值。\n§l加满§r：将玩家饥饿设置为最大值。
+wdl.gui.player.position.description=此选项可修改在下载的世界中玩家的位置。\n\n§l不做改变§r：使用下载结束时的玩家的位置\n§lXYZ§r：允许指定精确的坐标。
+wdl.gui.player.setPositionToCurrentPosition.description=将位置设置为您当前的位置。
+wdl.gui.player.positionTextBox.description=使用文本框指定精确的出生点坐标。\n\n此文本框指定的坐标为 %s 。
+
+wdl.gui.generator.title=World Downloader mod:: %s 的生成选项
```
Comment (Pokechu22) [inline share/src/main/resources/assets/wdl/lang/zh_CN.lang]: This looks like a typo; I assume it should only be 1 `:`?

**Review by Pokechu22 — COMMENTED** [review]
Thanks for doing this!

File: share/src/main/resources/assets/wdl/lang/zh_CN.lang (hunk 1 above)
Comment (ghost) [reply share/src/main/resources/assets/wdl/lang/zh_CN.lang]: alright

File: share/src/main/resources/assets/wdl/lang/zh_CN.lang (hunk 2 above)
Comment (ghost) [reply share/src/main/resources/assets/wdl/lang/zh_CN.lang]: yep, I have changed it.

**Comment by Pokechu22** [comment]
Merged; this will be in the next release (whenever that is).  Once again, thanks!

**Pull request merged**